tests/driver_netdev_common/Kconfig

```diff
@@ -0,0 +1,375 @@
+# Copyright (c) 2021 Gunar Schorcht
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config DRIVER_NETDEV_COMMON
+    bool
+    default y if TEST_KCONFIG
+
+    # Modules to include:
+    select MODULE_SHELL
+    select MODULE_SHELL_COMMANDS
+    select MODULE_PS
+
+    # gnrc is a meta module including all required, basic gnrc networking modules
+    select MODULE_GNRC
```

Comment:
> As @aabadie mentions, all these modules should be on their own files.

That is clear. It was just an attempt to test the `netdev` driver `Kconfigs` before the modeling of the network stack is complete. I was able to get green CI with them. As said, I thought that `tests/driver_netdev_common` with all these dummy definitions might help to model network related drivers and models step by step and remove them from `tests/driver_netdev_common` once they are modeled.

> I wonder if it is actually possible to use the driver without GNRC, in that case we could test the driver modelling alone.

Since the `feather-m0-wifi` board uses the driver as `netdev_default`, which in turn is activated by all GNRC applications, you will not be able to successfully compile these applications into CI without it being modeled somehow, even if it is a dummy definition to begin with.

Reply:
> Since the feather-m0-wifi board uses the driver as netdev_default, which in turn is activated by all GNRC applications, you will not be able to successfully compile these applications into CI without it being modeled somehow, even if it is a dummy definition to begin with.

But couldn't we model just the driver + netdev without GNRC? We would not test GNRC-based applications but at least the one from the driver, right? AFAIU `netdev_default` should not pull the stack.

Reply:
@gschorcht regarding this point, what I'm attempting for Ethernet (#17739) and IEEE 802.15.4 (#17789) is to change the tests to use the driver and netdev only, without pulling any stack.